src/3d/framegraph/qgsframegraph.h

```diff
@@ -333,7 +334,7 @@ class QgsFrameGraph : public Qt3DCore::QEntity
     bool mRenderCaptureEnabled = false;
 
     // holds renderviews according to their name
-    QMap<QString, std::shared_ptr<QgsAbstractRenderView>> mRenderViewMap;
+    mutable std::map<QString, std::unique_ptr<QgsAbstractRenderView>> mRenderViewMap;
```

Comment:
why `mutable` ? if it is because it is used from shadowRenderView() and forwardRenderView() which are const, then they probably should not be const methods - they are returning a non-const reference...

Reply:
yes this was the cause! I will remove the const and the mutable then!